docs/how-to/installation/aws.txt

```diff
@@ -205,20 +205,23 @@ Spinning up the Cluster
       -  10m (10 minutes)
 
    -  -  ``--max-dynamic-agents``
-      -  Maximum number of dynamic agent instances at one time.
+      -  Maximum number of dynamic agent instances in the default GPU
+         :ref:`resource pool <resource-pools>`.
       -  5
 
    -  -  ``--spot``
-      -  Use spot instances.
+      -  Use spot instances for the default CPU and GPU resource pools.
       -  False
 
    -  -  ``--spot-max-price``
 
-      -  A maximum price for the spot instances. If the price on the
-         spot market exceeds this value, Determined will not create new
-         instances. Optional
+      -  The maximum price to use when launching spot instances. If the
+         price on the spot market exceeds this value, Determined will
+         not create new instances. If no price is configured, the
+         maximum price will be the on-demand price for the configured
+         instance type and region.
 
-      -  The on-demand price for the instance type.
+      -  Not set
```

Comment:
I actually rather like having the default column say "the on-demand price", though maybe that's too long or something with "for the configured..."?

Reply:
I couldn't really find a way to phrase this that I was happy with, so I just kept the current text for now.